db/chain/cte.go

```diff
@@ -50,6 +50,9 @@ func (ec *ExpressionChain) renderctes(dst *strings.Builder) ([]interface{}, erro
 			return nil, errors.Wrapf(err, "rendering cte %s", name)
 		}
 		dst.WriteRune(')')
+		if i == len(ec.ctesOrder)-1 {
+			dst.WriteRune('.')
```

Comment:
@perrito666 I might be missing something, but just double checking, should this be `,` instead of `.` ?